.gitignore

```diff
@@ -0,0 +1,3 @@
+_build
+*.install
+.merlin
```

Comment:
maybe add `_opam` as well, for people using local switches?

Reply:
Aren't local switches typically used in user projects? Will calendar itself ever have a local switch?

Reply:
That's true. Maybe there's a nicer way to do things but sometimes i like to just go `opam switch create . <ocaml-version>` when trying out a new project so it installs everything i need to explore without messing with the my system wide switch. Its very useful when a given project has constraints about compiler version/libraries that don't play nice with one's global switches. But that being said, its fairly straightforward to add `_opam` in the `.git/info/exclude` so its probably not important to add it here as well.

Reply:
One reason that I noticed this was also ignoring these files in any other tool like `Ctrl-P`, `fzf`, `ripgrep` etc which respect the ignore files.

Reply:
That's a cool trick. You can also add `_opam` to your global `.gitignore`, I think those tools respect it.